src/pages/admin/company-profile/components/company-profile-requisites-tab/CompanyProfileRequisitesTab.scss

```
@@ -0,0 +1,56 @@
+            font-size: typography.$small-size;
+            line-height: typography.$small-height;
+            letter-spacing: typography.$small-spacing;
+            font-weight: typography.$fw-medium;
+            color: #374151;
+
+            .required {
+                color: colors.$error;
+            }
+        }
+
+        error {
```

Stylelint (17.4.0)

[error] Unexpected unknown type selector "error" (selector-type-no-unknown)

(selector-type-no-unknown)